R/uniDimBayesianPreCalc.R

```diff
@@ -62,7 +71,7 @@
     model[["n"]] <- nrow(dataset)
 
     # check if any items correlate negatively with the scale
-    model[["footnote"]] <- .checkLoadings(dataset, options[["variables"]])
+    model[["footnote"]] <- gettextf("%s %s", model[["footnote"]], .checkLoadings(dataset, options[["variables"]]))
```

Comment:
so, this implies that translators will be asked to translate
```r
%s %s
```
from one language to another. Do you think that makes sense? I'd use `sprintf` instead.